newrelic-php-agent/templates/daemonset.yaml

```diff
@@ -0,0 +1,33 @@
+{{- if .Values.deamonset.enabled }}
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: {{ template "newrelic-php-agent.fullname" . }}
+  labels:
+    app: {{ template "newrelic-php-agent.name" . }}
+    chart: {{ template "newrelic-php-agent.chart" . }}
+    heritage: {{ .Release.Service | quote }}
+    release: {{ .Release.Name | quote }}
```

Comment:
```
metadata:
  name: {{ include "foo.fullname" . }}
  labels:
{{ include "foo.labels" . | indent 4 }}
```

The way to specify labels for charts generated with the latest Helm is `{{ include "chart-name.labels" . | indent 4 }}`.

Reply:
@cw-sakamoto 
Do you want to match the chart of other chatwork/charts?
Or do you want to match the latest helm?

Reply:
I'm not particular to chart labels, so either is fine😀 If anything, latest helm ?

Reply:
@cw-sakamoto 
:+1: 

@cw-niinuma 
So let's match the latest helm.